### ISSUE TITLE
koverHtml Report - FileNotFoundException: testDebugUnitTest.ic.smap

### ISSUE THREAD

**Issue by AshStefanOltmann** [Kotlin/kotlinx-kover#50]
Running `koverReport` some file could not be found and that break the build.

Consider turning this into a warning that the report could not be created because something is missing.

```
java.io.FileNotFoundException: /home/runner/work/myapp/shared/build/kover/shared/testDebugUnitTest.ic.smap (No such file or directory)
	at java.base/java.io.FileInputStream.open0(Native Method)
	at java.base/java.io.FileInputStream.open(FileInputStream.java:211)
	at java.base/java.io.FileInputStream.<init>(FileInputStream.java:153)
	at com.intellij.rt.coverage.instrumentation.SaveHook.loadSourceMapFromFile(SaveHook.java:136)
	at com.intellij.rt.coverage.instrumentation.SaveHook.loadAndApplySourceMap(SaveHook.java:121)
	at com.intellij.rt.coverage.report.Reporter.getProjectData(Reporter.java:72)
	at com.intellij.rt.coverage.report.Reporter.createHTMLReport(Reporter.java:103)
	at com.intellij.rt.coverage.report.Main.main(Main.java:47)
```

Running only `koverXmlReport` works, so `koverHtmlReport` seems to be broken.

**Comment by shanshin** [comment]
Hi,
could you send a reproducer or a link to the project?

**Comment by shanshin** [comment]
Duplicate #46

**Comment by AshStefanOltmann** [comment]
Closed as this is indeed a duplicate of #46
